taskcluster/scripts/toolchain/cross-compile-setup.sh

```diff
@@ -25,7 +25,11 @@ tooltool.py \
   --manifest="/builds/worker/checkouts/src/libs/macos-cc-tools.manifest" \
   fetch
 
+popd || exit
+
+# It's important that we execute rustup in the checkout directory, so that it can see
+# any `rust-toolchain` override file that might be present.
+rustup --version
 rustup target add x86_64-apple-darwin
 rustup target add x86_64-pc-windows-gnu
```

Comment:
These targets are really only useful in CI, so I opted to continue installing them here explicitly rather than making everyone get them by default.